packages/storage-plugin/tests/storage.plugin.spec.ts

```diff
@@ -66,8 +68,6 @@ describe('NgxsStoragePlugin', () => {
   });
 
   it('should save data to localstorage', () => {
-    localStorage.setItem('@@STATE', JSON.stringify({ counter: { count: 100 } }));
```

Comment:
Why remove?

Reply:
basically because thats already covered with the "should initialize"-test 

so it fails because of that init-test, and not because writing doesnt work